protos/piecestore/piece_store.proto

```diff
@@ -0,0 +1,60 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+syntax = "proto3";
+
+package piecestoreroutes;
+
+service PieceStoreRoutes {
+  rpc Piece(PieceId) returns (PieceSummary) {}
+
+  rpc Retrieve(PieceRetrieval) returns (stream PieceRetrievalStream) {}
+
+  rpc Store(stream PieceStore) returns (PieceStoreSummary) {}
+
+  rpc Delete(PieceDelete) returns (PieceDeleteSummary) {}
+
+}
+
+message PieceStore {
+  string id = 1;
+  int64 size = 2;
+  int64 ttl = 3;
+  int64 storeOffset = 4;
+
+  bytes content = 5;
+}
+
+message PieceId {
+  string id = 1;
+}
+
+message PieceSummary {
+  string id = 1;
+  int64 size = 2;
+  int64 expiration = 3;
```

Comment:
what happens for data that has no expiration? sometimes data doesn't expire ever

Reply:
Shoul we handle a blank ttl in the server, or client?

Reply:
i think both the server and the client need to handle that case. essentially, some data we're saying "only store this file for 10 days" and some data we're saying "please store this forever. never remove it"